lib/db/schema/patch-090-091.sql

```diff
@@ -0,0 +1,243 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('90');
+
+-- The `profileChangedAt` column may or may not exist in production env depending
+-- on how migration 87 went. If it doesn't exist, then this statement will error because
+-- you can not drop non-existent column in MySQL.
+ALTER TABLE accounts DROP COLUMN profileChangedAt,
```

Comment:
This was one tricky bit that I wasn't sure how to approach. The column maybe or maybe exist depending if migration 87 was performed. MySQL will yell if you attempt to drop a column that doesn't exist. There doesn't appear to be a `DROP IF EXISTS` option for columns either.

Could this statement be tweaked to not run during our migration?

Reply:
I think this is fine, @jrgm will just remove this statement when running the migration in production.